src/docs/development/platform-integration/add-to-app-android/add-splash-screen.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Adding a splash screen and launch screen to an Android app
+short-title: Add a splash screen
+description: Learn how to add a splash screen and launch screen to your Android app.
+---
+
+{% asset
+development/platform-integration/add-to-app-android/add-splash-screen/splash-screens_header.png
+class="mw-100" alt="Add Splash Screen Header" %}
+
+The beginning of a Flutter experience requires a brief wait while Dart
+initializes. Additionally, a full-Flutter app requires a standard Android app
+initialization time, too. Flutter supports the display of a "launch screen"
+while your Android app initializes, and also supports the display of a "splash
+screen" while your Flutter experience initializes. This guide teaches you how to
+use launch screens and splash screens in an Android app with Flutter.
+
+{{site.alert.note}} 
+  Strategies are available to minimize wait time related to Flutter 
+  initialization. Consider [pre-warming a FlutterEngine](), and 
+  [re-using a FlutterEngine throughout your app](), to avoid most wait time.
+{{site.alert.end}} 
+
+## Android Launch Screen
+
+Every Android app requires initialization time while the operating system sets
+up the app's Process. Android provides the concept of a [launch screen] to
+display a `Drawable` while the app is initializing.
+
+[launch screen]: {{site.android-dev}}/topic/performance/vitals/launch-time#themed
+
+Flutter provides support for displaying an Android launch screen before showing
+a `FlutterActivity`. Here are instructions to display an Android launch screen:
+
+### Define a launch theme
+
+In `styles.xml`, define a theme whose `windowBackground` is set to the
+`Drawable` that should be displayed as the launch screen.
+
```

Comment:
This is still in the default flutter create template right? If so, mention it

Reply:
done